package.json

```diff
@@ -40,7 +40,6 @@
     "@types/sinon": "~5.0",
     "@types/sinon-chai": "~3.2",
     "basichtml": "~0.20",
-    "buble": "0.19.4",
```

Comment:
Will this not break the docs anymore?
Also, with this change, do we not need to update the lock file(s)?

Reply:
npm did not wanted to update package-lock. yarn.lock is updated. Yeah they fixed it 👍